src/controller/CHIPDeviceController.cpp

```diff
@@ -1246,12 +1246,12 @@ CHIP_ERROR DeviceCommissioner::IssueNOCChain(const ByteSpan & NOCSRElements, Nod
     // Note: we don't have attestationSignature, attestationChallenge, DAC, PAI so we are just providing an empty ByteSpan
     // for those arguments.
     return mOperationalCredentialsDelegate->GenerateNOCChain(NOCSRElements, ByteSpan(), ByteSpan(), ByteSpan(), ByteSpan(),
-                                                             ByteSpan(), callback);
+                                                             ByteSpan(), ByteSpan(), callback);
 }
 
 CHIP_ERROR DeviceCommissioner::ProcessCSR(DeviceProxy * proxy, const ByteSpan & NOCSRElements,
-                                          const ByteSpan & AttestationSignature, const ByteSpan & dac, const ByteSpan & pai,
-                                          const ByteSpan & csrNonce)
+                                          const ByteSpan & AttestationSignature, const ByteSpan & attestationElements,
```

Comment:
On CSRResponse, you don't have `attestationElements`, you have `NOCSRElements`

Reply:
reverted signature change